src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/intro";

```

Comment:
The `DB_URL` is incomplete. You need to specify the full JDBC URL to connect to the database, including the database name and any other required parameters for the connection.